bazaarvoice/astyanax
# PD-194386: fix the SonarQube Security Hotspots

## Jira Issue
[PD-194386](https://bazaarvoice.atlassian.net/browse/PD-194386)

## What Are We Doing Here?
Fix the SonarQube Security Hotspots

## How to Test and Verify

1. Check out this PR
2. Run: `mvn clean install -U -Ddependency-check.skip=true`
3. Run: `mvn -e -Ddependency-check.skip=true verify`
4. Check if the tests passed successfully


## Risk Level
`Low`

## Required Testing
`Smoke`, `Regression`
